doc/changelog/10-coqide/19417-coqide-preferences.rst

```diff
@@ -0,0 +1,22 @@
+- **Changed:**
+  Improved Preferences dialog: larger margins,
+  tree of categories, sections in the categories,
+  spin buttons for numbers, preservation of the last
+  selected category, and more
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Fixed:**
+  All preferences are now applied after clicking Apply or OK
+  rather than immediately
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Fixed:**
+  Changing the allowed modifiers in the Shortcuts panel of
+  the Preferences dialog now immediately updates the available
+  modifiers for the listed menus
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
+- **Added:**
+  Preference setting for unjustified conclusions background color
+  (`#19417 <https://github.com/coq/coq/pull/19417>`_,
+  by Sylvain Chiron).
```

Comment:
It seems odd to have both an Appearance tab and "Editor appearance" on the Editor tab.  Maybe the editor ones could move to the Appearance tab?  Maybe move the window side ones to the bottom of the panel?

Reply:
If it is really a problem of reusing the same word, I think the best thing to do is to use another word in “Editor appearance”, e.g. “Editor display”, “Editor view”, “Editor layout”.